src/app/CASEClient.h

```diff
@@ -28,11 +28,11 @@ class CASEClient;
 
 struct CASEClientInitParams
 {
-    SessionManager * sessionManager                     = nullptr;
-    SessionResumptionStorage * sessionResumptionStorage = nullptr;
-    Messaging::ExchangeManager * exchangeMgr            = nullptr;
-    FabricInfo * fabricInfo                             = nullptr;
-    Credentials::GroupDataProvider * groupDataProvider  = nullptr;
+    SessionManager * sessionManager                             = nullptr;
+    AbstractSessionResumptionStorage * sessionResumptionStorage = nullptr;
```

Comment:
We don't use the prefix `Abstract` anywhere. It's more that `SessionResumptionStorage` concrete impl should have been named `DefaultSessionResumptionStorage` or something

Reply:
I'm open to any naming suggestions.

Currently we have SessionResumptionStorage, which is a partial implementation, and SimpleSessionResumptionStorage which inherits this and is a full implementation.

I'm not sure what three names would be best, but I don't think SessionResumptionStorage can be Default because it has pure virtual methods.

Reply:
On second thought, I think DefaultSessionResumptionStorage is fine.  It's the default 'strategy', which has an index and bidirectional maps, and some implementation.

I'll change naming accordingly.

Reply:
Done.